content/post/kubedb-v2022.10.18/index.md

```diff
@@ -0,0 +1,785 @@
+```
+Example of a PerconaXtraDBAutoscaler object, where we want to autoscale the storage resources.
+```yaml
+apiVersion: autoscaling.kubedb.com/v1alpha1
+kind: PerconaXtraDBAutoscaler
+metadata:
+  name: px-auto
+  namespace: demo
+spec:
+  databaseRef:
+    name: pxc
+  opsRequestOptions:
+    apply: Always
+    timeout: 10m
+  storage:
+    perconaxtradb:
+      trigger: "On"
+      usageThreshold: 60 # If PVC usage percentage is less than the usageThreshold, we don't want to scale the storage.
+      scalingThreshold: 50 # If the usageThreshold criteria is satisfied, we want to scale PVC storage by the specified scalingThreshold.
+      expansionMode: "Offline" # possible values : `Offline` & `Online`.
+```
+
+
+## MongoDB
+We introduce MongoDB hidden members in this latest release. A hidden member in mongoDB is a part of replica-set.
+It maintains the copy of the primary data set, but remains invisible to client applications. They are good for workloads with different usage patterns.
+Like, You are using inMemory databases for your primary & electable secondaries (to improve latency issue), but at the same time, you also want your data not to be lost in time of pod restart, then Hidden member is the solution for you. 
+
+Here is a sample hidden member enabled mongoDB sharded cluster example:
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MongoDB
+metadata:
+  name: mongo-sh
+  namespace: demo
+spec:
+  version: "4.4.6"
+  shardTopology:
+    configServer:
+      replicas: 3
+      storage:
+        resources:
+          requests:
+            storage: 5Gi
+        storageClassName: "linode-block-storage"
+    mongos:
+      replicas: 2
+    shard:
+      replicas: 2
+      shards: 2
+      storage:
+        resources:
+          requests:
+            storage: 5Gi
+        storageClassName: "linode-block-storage"
+  terminationPolicy: WipeOut
+  arbiter: {}
+  hidden:
+    podTemplate:
+      spec:
+        resources:
+          requests:
+            cpu: "500m"
+            memory: "500Mi"
+    replicas: 2
+    storage:
+      storageClassName: "linode-block-storage"
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 15Gi
+```
+
+
+## MySQL
+### New Version Support
+We have added the most recent MySQL versions.
+Now you can create a Standalone , Group Replication, Innodb Cluster, Read Replica and Semi-sync Cluster with `MySQL 8.0.31`
+You can get started with `mysql:8.0.31` by applying this simple yaml
+### BugFix and Improvement
+Several bugs in mysql ops request including mysql version upgrading and reconfigure TLS and Custom configuration are fixed.
+OpsRequest for innodb cluster were improved. And in this release innodbCluster is more stable in provisioning and failover.
+Now kubedb managed MySQL instance can automatically recover when some group member has extra transaction on it. It will clone from the Primary instance to recover.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
```

Comment:
move this yaml to MySQL New version support section

Reply:
Good one. Updated